src/types/viewport.cpp

```diff
@@ -365,11 +365,12 @@ bool Viewport::DecrementInBoundsCircular(til::point& pos) const noexcept
 // -  This is so you can do s_CompareCoords(first, second) <= 0 for "first is left or the same as second".
 //    (the < looks like a left arrow :D)
 // -  The magnitude of the result is the distance between the two coordinates when typing characters into the buffer (left to right, top to bottom)
+#pragma warning(suppress : 4100)

```

Comment:
???

Reply:
It's dumb. `allowEndExclusive` is now a param that's not referenced anywhere... but like, it is, it's just hidden behind an `assert`.